modules/common/affinity/affinity.go

```
@@ -53,4 +58,46 @@ func DistributePods(
 			},
 		},
 	}
+	// patch the default affinity Object with the data passed as input
+	if overrides != nil {
+		patchedAffinity, _ := toCoreAffinity(defaultAffinity, overrides)
```

can we ignore the error?

---

mmm we could backpropagate the resulting error, I agree

---

Done